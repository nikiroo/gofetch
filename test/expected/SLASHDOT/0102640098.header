0Software Developers Are Now More Valuable To Companies Than Money, Says Survey (cnbc.com)	null/SLASHDOT/0102640098		70
i  Thursday September 06, 2018 @11:30PM (BeauHD)
i  from the new-breed-of-corporate-leaders dept.
i
i    An anonymous reader quotes a report from CNBC:
i    
i    >   As   our  global  economy  increasingly  comes  to  run  on
i    technology-enabled  rails  and  every  company  becomes  a tech
i    company,  demand  for  high-quality software engineers is at an
i    all-time  high.  A  recent study from [1]Stripe and Harris Poll
i    found  that  61 percent of C-suite executives believe access to
i    developer  talent is a threat to the success of their business.
i    Perhaps  more  surprisingly  --  as  we mark a decade after the
i    financial  crisis  --  [2]this  threat  was  even  ranked above
i    capital   constraints   .   And   yet,   despite   being   many
i    corporations'  most  precious  resource,  developer talents are
i    all  too  often  squandered. Collectively, companies today lose
i    upward  of $300 billion a year paying down "technical debt," as
i    developers   pour  time  into  maintaining  legacy  systems  or
i    dealing  with  the  ramifications  of  bad  software.  This  is
i    especially  worrisome,  given  the  outsized  impact developers
i    have  on  companies'  chances  of  success. Software developers
i    don't  have a monopoly on good ideas, but their skill set makes
i    them  a  uniquely  deep  source of innovation, productivity and
i    new  economic  connections. When deployed correctly, developers
i    can  be  economic multipliers -- coefficients that dramatically
i    ratchet  up  the  output  of  the  teams and companies of which
i    they're a part.
i    
i    
i    
i    [1] https://stripe.com/reports/developer-coefficient-2018
i    
i    [2] https://www.cnbc.com/2018/09/06/companies-worry-more-about-
i    access-to-software-developers-than-capital.html
i
