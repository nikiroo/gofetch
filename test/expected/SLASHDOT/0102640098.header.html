<!DOCTYPE html>
<html>
<head>
  <meta http-equiv='content-type' content='text/html; charset=utf-8'>
  <meta name='viewport' content='width=device-width, initial-scale=1.0'>
  <style type='text/css'>
    body { margin: 1em 15%; }
  </style>
</head>
<body>
<div class='story-header'>
	<h2><a href='0102640098.html'>Software Developers Are Now More Valuable To Companies Than Money, Says Survey (cnbc.com)</a></h2>
	<div class='details'>(Thursday September 06, 2018 @11:30PM (BeauHD) 
from the new-breed-of-corporate-leaders dept.)</div>
	<br/>
	<div class='content' style='text-align: justify'>
		An anonymous reader quotes a report from CNBC: As our global economy increasingly comes to run on technology-enabled rails and every company becomes a tech company, demand for high-quality software engineers is at an all-time high. A recent study from Stripe and Harris Poll found that 61 percent of C-suite executives believe access to developer talent is a threat to the success of their business. Perhaps more surprisingly -- as we mark a decade after the financial crisis -- this threat was even ranked above capital constraints. And yet, despite being many corporations&#x27; most precious resource, developer talents are all too often squandered. Collectively, companies today lose upward of $300 billion a year paying down &quot;technical debt,&quot; as developers pour time into maintaining legacy systems or dealing with the ramifications of bad software. This is especially worrisome, given the outsized impact developers have on companies&#x27; chances of success. Software developers don&#x27;t have a monopoly on good ideas, but their skill set makes them a uniquely deep source of innovation, productivity and new economic connections. When deployed correctly, developers can be economic multipliers -- coefficients that dramatically ratchet up the output of the teams and companies of which they&#x27;re a part.
	</div>
<hr/>
</div>
</body>
